doc/developers/contributing.rst

```diff
@@ -952,13 +952,14 @@ The easiest and recommended way to accomplish this is to
 All logic behind estimator parameters,
 like translating string arguments into functions, should be done in ``fit``.
 
-Parameters can be passed to the ``fit`` method. Generally, these parameters
+While it is possible to pass parameters into the ``fit`` method, these
 should be restricted to variables that need to be sliced during
-cross-validation. These variables would be either arrays with shape=[N] or
-array-like with shape=[N,] where N is the number of samples. An example of
+cross-validation. These variables would be either arrays with shape=[N] where N=n_samples
```

Comment:
I think there are many ways to express the same idea here. I like the
method you're suggesting as it's very clear and concise. I'll make this
change today.

On Sep 6, 2016 3:03 PM, "Andreas Mueller" <notifications@github.com> wrote:

> In doc/developers/contributing.rst
> <https://github.com/scikit-learn/scikit-learn/pull/7156#discussion_r77698216>
> :
>
> >  should be restricted to variables that need to be sliced during
> > -cross-validation. These variables would be either arrays with shape=[N] or
> > -array-like with shape=[N,] where N is the number of samples. An example of
> > +cross-validation. These variables would be either arrays with shape=[N] where N=n_samples
>
> I think we only require shape[0] == n_samples. Why not just These would
> be arrays withshape[0] == n_samples``?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/scikit-learn/scikit-learn/pull/7156/files/43fdd093b0bf14a22d7371334503949d61dfb491..01ccec3d297b421d86876d15f35df65aef11b511#r77698216>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKNQ_CnJUt4oMYWJLxQR-DtpLDO9IfjDks5qnbkTgaJpZM4JeZ4l>
> .
>